test/lib/command-mocks.js

```diff
@@ -40,6 +40,38 @@ module.exports = {
     }, true);
   },
 
+
+  deleteCookie() {
```

Comment:
I don't think we need to add command mocks, as `fakedriver` already have `deleteCookie` command
https://github.com/gravityvi/nightwatch/blob/eb0f2eb75c1edba5b72b83e65d813f0f081c8890/test/lib/fakedriver.js#L143-L143

Reply:
Fake driver is only being used while running protocol tests if we don't add mocks the request is giving `404`.

Response
```
  Response 404 DELETE /wd/hub/session/1352110219202/cookie/other_cookie (2ms)
{}
```

Reply:
Yes, it's a more complete test. The protocol test is only checking if the right method is used.